2022-RapidReact/src/main/java/frc/robot/commands/GrabBalls.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.Intake;
+
+public class GrabBalls extends CommandBase {
+  /** Creates a new GrabBalls. */
+  private final Intake r_intake;
+
+  public GrabBalls(Intake param_intake) {
+    // Use addRequirements() here to declare subsystem dependencies.
+  r_intake = param_intake;
+  addRequirements(r_intake);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    r_intake.grabBalls(Constants.GB_VELOCITY);
```

Comment:
isso me parece que vai dar o erro de que falta ser atualizado esse valor, aquele not updated enough. o lugar certo de executar algo é no execute